rmw_connext_shared_cpp/CMakeLists.txt

```diff
@@ -56,3 +56,6 @@ install(
   LIBRARY DESTINATION lib
   RUNTIME DESTINATION bin
 )
+
+ament_index_register_resource("rmw_output_filter"
+  CONTENT "RTI Data Distribution Service\nExpires on")
```

Comment:
Should this information be passed through the index or just be exposed in a CMake variable / function?